docs/src/main/asciidoc/mongodb-panache-guide.adoc

```diff
@@ -0,0 +1,525 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Simplified MongoDB with Panache
+
+include::./attributes.adoc[]
+:config-file: application.properties
+
+MongoDB is a well known NoSQL Database that is widely used, but using its raw API can be cumbersome as you need to express your entities and your queries as a MongoDB link:https://mongodb.github.io/mongo-java-driver/3.11/bson/documents/#document[`Document`].
+
+MongoDB with Panache provides active record style entities (and repositories) like you have in link:hibernate-orm-panache-guide.html[Hibernate ORM with Panache] and focuses on making your entities trivial and fun to write in Quarkus.
+
+It is build on top of the link:mongo-guide.html[MongoDB Client] extension.
+
+WARNING: This extension is still experimental, feedback are welcome on the [mailing list](https://groups.google.com/d/forum/quarkus-dev) or as issue in our [github issues tracker](https://github.com/quarkusio/quarkus/issues).
+
+== First: an example
+
+Panache allows you to write your MongoDB entities like this:
+
+[source,java]
+--
```

Comment:
You should use `----` for source blocks.

Reply:
Done for all blocks.
It was the same on the hibernate with Panache guide. It will be part of a PR for this guide to align some corrections/typos between the two guides.